tests/chocolatey-tests/commands/choco-config.Tests.ps1

```diff
@@ -125,6 +125,15 @@ Describe "choco config" -Tag Chocolatey, ConfigCommand {
             $Output.Lines | Should -Contain "Use choco apikey to interact with API keys."
         }
 
+        It "Displays deprecation note about sources and features" {
+            $pattern = @(
+                "Starting in v2.0.0, the `config list` command will only list the"
+                " configuration values, instead of also listing the configured features"
+                " and sources."
+            ) -join "\r?\n"
+            $Output.String | Should -Match "Starting in v2.0.0, the `config list` command will only list the\r?\n configuration values, instead of also listing the configured features\r?\n and sources."

```

Comment:
Is there a reason this sets up a pattern and then doesn't use it? Also, correct me if I'm wrong, but doesn't `Invoke-Choco` trim the lines before joining them onto the `String` property?

Reply:
Huh, oops. Let me double check on that and update this.

Reply:
![image](https://user-images.githubusercontent.com/32407840/218772459-78d18f69-b58c-4dff-8a9e-79f3ed287ae6.png)
Invoke-Choco currently only trims lines for `.Lines`, not `.String` it seems